cmd/podman/system/unshare.go

```diff
@@ -33,6 +34,7 @@ func init() {
 	})
 	flags := unshareCommand.Flags()
 	flags.SetInterspersed(false)
+	flags.BoolVar(&unshareOptions.RootlessCNI, "rootless-cni", false, "Join the rootless-cni namespace")
```

Comment:
Suggest rewording description to "Join the rootless network namespace used for CNI networking"

Reply:
Done